tests/macros/macro_lib_d.html

```diff
@@ -0,0 +1,4 @@
+{% macro inner_impl() %}hello{% endmacro inner_impl -%}
+{# https://github.com/Keats/tera/issues/532
+   Try changing self:: to lib_d:: to pass one test and fail another. #}
```

Comment:
Like in the first commit, this would get the first test to pass but not the second that uses no inheritance.